apps/web/src/components/AccountSelectorModal/AccountSelectorPopover.test.tsx

```diff
@@ -0,0 +1,146 @@
+import { useDynamicDisclosureContext } from "@umami/components";
+import {
+  type ImplicitAccount,
+  type LedgerAccount,
+  type MnemonicAccount,
+  type MultisigAccount,
+  type SecretKeyAccount,
+  type SocialAccount,
+} from "@umami/core";
+import { useSelectedNetwork } from "@umami/state";
+
+import { AccountInfoModal } from "./AccountInfoModal";
+import { AccountSelectorPopover } from "./AccountSelectorPopover";
+import { RemoveAccountModal } from "./RemoveAccountModal";
+import { RenameAccountPage } from "./RenameAccountModal";
+import { act, render, screen, userEvent } from "../../testUtils";
+
+jest.mock("@umami/components", () => ({
+  ...jest.requireActual("@umami/components"),
+  useDynamicDisclosureContext: jest.fn(),
+}));
+jest.mock("@umami/state", () => ({
+  ...jest.requireActual("@umami/state"),
+  useSelectedNetwork: jest.fn(),
+}));
+
+const mockAccount = {
+  label: "Test Account",
+  address: { pkh: "tz1TestAddress" },
+  type: "social",
+} as ImplicitAccount;
+
+const mockNetwork = {
+  tzktExplorerUrl: "https://tzkt.io",
+};
+
+describe("AccountSelectorPopover", () => {
+  beforeEach(() => {
+    (useDynamicDisclosureContext as jest.Mock).mockReturnValue({
```

Comment:
it's already available to you with the `dynamicDisclosureContextMock`. please check testUtils.tsx